biojava-core/src/main/java/org/biojava/nbio/core/sequence/features/AbstractFeature.java

```diff
@@ -285,8 +285,17 @@ public void setQualifiers(HashMap<String, Qualifier> qualifiers) {
 
 	@Override
 	public void addQualifier(String key, Qualifier qualifier) {
-		// TODO Auto-generated method stub
-		Qualifiers.put(key, qualifier);
+		// Check for key. Update list of values
```

Comment:
I am happy to go either way. However, when I try to use the generic `Map` and`List` I run into errors like the following: 

![screen shot 2015-07-20 at 11 59 11 am](https://cloud.githubusercontent.com/assets/1260667/8780435/ea2dfeb6-2ed6-11e5-86e6-44e35fd8cfd0.PNG)
.

So I am not really sure how to implement the changes as you suggest.